src/blocks/intermediary/data.rs

```diff
@@ -39,25 +45,29 @@ impl<'raw> ModernBlockList<'raw> {
 
 /// Compact way of identifying block data.
 ///
-/// Only if the block has one ore more properties, the [`ModernBlockData::kinds`]
-/// will be serialized. If the block has more than one blockstate,
-/// a default_id field will be serialized as well.
+/// Only if the block has one ore more properties, the
+/// [`ModernBlockData::kinds`] will be serialized. If the block has more than
+/// one blockstate, a default_id field will be serialized as well.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct ModernBlockData<'raw> {
-    #[serde(borrow, skip_serializing_if = "LinkedHashMap::is_empty", rename = "properties")]
-    kinds: LinkedHashMap<&'raw str, PropertyValue<'raw>, RandomState>,
+    #[serde(borrow, skip_serializing_if = "Option::is_none", rename = "properties")]
+    kinds: Option<LinkedHashMap<&'raw str, PropertyValue<'raw>, RandomState>>,
```

Comment:
Why change this?

Reply:
Otherwise I would get `missing field `properties` at line 184 column 5` which is the first block in the lists of blocks. I assumed it thought it wasn't okay that

Reply:
I got `missing field `properties` at line 184 column 5` which is where the first block in the block list is. I assumed this was because it expected the list instead of assuming no list meant empty list. The Option helped.

Reply:
Then I would recommend trying out using the default attribute, that keeps things a lot more dev-friendly.